staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -63,7 +63,6 @@ func (c *fakeObjectConvertor) Convert(in, out, context interface{}) error {
 		out, err = c.converter.Convert(typedValue, c.apiVersion)
 		return err
 	}
-	out = in
```

Comment:
@apelisse what was this line trying to do?

Reply:
I think this means: make the conversion a no-op, but that's needed if you want the output to be equal to the input.